docs/api.md

```diff
@@ -1380,6 +1384,157 @@ Exposes the Pino package version. Also available on the logger instance.
+stream.write('helloworld')
+stream.end()
+
+stream.on('error', (err) => {
+  console.log(err) // Unexpected token h in JSON at position 0
+})
+```
+
+<a id="pino-test-once"></a>
+### `pino.test.once(stream, expected, is) => Promise<void>`
+Assert that a single log is expected.
+
+```js
+const pino = require('pino')
+
+const stream = pino.test.sink()
+const instance = pino(stream)
+
+instance.info('hello world')
+
+const expected = { msg: 'hello world', level: 30 }
+await pino.test.once(stream, expected) // doesn't throw a diff error
+await pino.test.once(stream, { msg: 'bye world', level: 30 }) // throw a diff error
+
+// OR logging an object
+instance.info({ hello: 'world', hi: 'world' })
+await pino.test.once(stream, { hello: 'world', hi: 'world', level: 30 }) // doesn't throw a diff error
+await pino.test.once(stream, { hello: 'world', level: 30 }) // throw a diff error
+
+// OR using a custom assert function
+function is (received, expected, msg) {
+  if (received.msg !== expected.msg) {
+    throw new Error(`expected msg ${expected.msg} doesn't match the received one ${received.msg}`)
+  }
+}
+
+await pino.test.once(stream, expected, is) // doesn't throw an error
+await pino.test.once(stream, { msg: 'bye world', level: 30 }, is) // throw an error
+```
+
+<a id="pino-test-consecutive"></a>
+### `pino.test.consecutive(stream, expected, is) => Promise<void>`
+Assert that consecutive logs are expected.
+
+```js
+const pino = require('pino')
+
+const stream = pino.test.sink()
+const instance = pino(stream)
+
+instance.info('hello world')
+instance.info('hi world')
+
+const expected = [
+  { msg: 'hello world', level: 30 },
+  { msg: 'hi world', level: 30 }
+]
+
+await pino.test.consecutive(stream, expected) // doesn't throw a diff error
+await pino.test.consecutive(stream, [{ msg: 'bye world', level: 30 }]) // throw a diff error
+
+// OR logging an object
+instance.info({ hello: 'world' })
+instance.info({ hi: 'world' })
+await pino.test.consecutive(stream, [
+  { hello: 'world', level: 30 },
+  { hi: 'world', level: 30 }
+])
+
+// OR using a custom assert function
+function is (received, expected, msg) {
+  if (received.msg !== expected.msg) {
+    throw new Error(`expected msg ${expected.msg} doesn't match the received one ${received.msg}`)
+  }
+}
+
+await pino.test.consecutive(stream, expected, is) // doesn't throw an error
+await pino.test.consecutive(stream, [{ msg: 'bye world', level: 30 }], is) // throw an error
+```
+
+Both `pino.test.once` and `pino.test.consecutive` internally
+- assert log message `time` is less than or equal to the current time
+- assert log message `pid` matches the current process id
+- assert log message `hostname` matches the current hostname
+- uses the default `deepStrictEqual` assert function of the `node:assert` module.
+
+See
+* [Transform](https://nodejs.org/api/stream.html#class-streamtransform)
+* [split2](https://github.com/mcollina/split2#readme)
```

Comment:
Why would someone need to reference these external resources? What is this "See" section referring to?

Reply:
no longer needed; let me remove it.